### ISSUE TITLE
you can make video install  and tutorial  duel links bot (steam version) step by step ?

### ISSUE THREAD

**Issue by asdasd888** [will7200/Yugioh-bot#35]
you can make video install  and tutorial  duel links bot (steam version) step by step ?

**Comment by koreplay** [comment]
Have you seen the gif its pretty straight forward.

**Comment by asdasd888** [comment]
i never using python 
![ss](https://user-images.githubusercontent.com/7657198/35187834-3b36f2c4-fe5d-11e7-8f4f-6aa536d8b000.jpg)

**Comment by curtiola** [comment]
I could be wrong, but in the Yugioh-bot folder, I'm pretty sure you just have to:
- right click config.ini
- open with notepad
- where it says "provider = Nox" change it to "provider = Steam"
- then save & close

**Comment by asdasd888** [comment]
i can't   git clone https://github.com/will7200/Yugioh-bot

**Comment by curtiola** [comment]
Use Git Bash.
https://git-scm.com/downloads

**Comment by asdasd888** [comment]
i can't $ pip install -r requirements.txt (or use conda if using)
           $ pip install -r install_requirements.txt
![sqs](https://user-images.githubusercontent.com/7657198/35217406-c26d5df0-ff9d-11e7-9e34-10a139ed5b11.jpg)

**Comment by curtiola** [comment]
It appears you didn't install Python properly, you needed to choose the custom installation option & check the box that says something similar to: "Add install path to System Environment".
Easiest way to solve is to reinstall it.

**Comment by will7200** [comment]
Pip is not on your path. If you used conda and didn't add to path then you need to open the environment from their program. You can also install the executable I made for version 0.4.2 but that is about 8 days old and there have been fixes since then. I will try and upload the latest. https://github.com/will7200/Yugioh-bot/releases

**Comment by asdasd888** [comment]
how to change path cd in Git Bash

**Comment by will7200** [comment]
Yes the .exe is located under the releases. It is in the zip with the exe named dlbot

**Comment by asdasd888** [comment]
dlbot.exe stop working
![1](https://user-images.githubusercontent.com/7657198/35404187-a6f19a2a-0234-11e8-816c-28255c65c7bf.jpg)

**Comment by will7200** [comment]
Did you try dlbot.exe gui -s?

**Comment by asdasd888** [comment]
I do not know how to use

**Comment by will7200** [comment]
Open a command prompt in that folder where dlbot.exe is located and run
```
.\dlbot.exe gui -s
```
Also make sure to redownload the latest release as the last one had a build error for the gui

**Comment by asdasd888** [comment]
![1](https://user-images.githubusercontent.com/7657198/35479677-af762eb6-042f-11e8-96c6-8de48f4fdbcd.jpg)

**Comment by will7200** [comment]
Did you re-download the latest release? If so I need to tag it correctly and rebuild it.

**Comment by asdasd888** [comment]
you can make video for newbie first step (detail)

**Comment by faber5594** [comment]
hi, first of all thanks for your work guys! then I got the same two errors of asdasd888 but with others lines, hope this could help you finding the solution. Thanks.
![immagine](https://user-images.githubusercontent.com/35998149/35622699-0fbf7920-0689-11e8-97ea-029a3e4148a0.jpg)

**Comment by curtiola** [comment]
you need to install pywin32, make sure to get the right one for your python installation.
https://github.com/mhammond/pywin32/releases

**Comment by faber5594** [comment]
@curtiola I installed it now it don't show the win32gui problem anymore, but it looks like it still doesn't work.
![immagine](https://user-images.githubusercontent.com/35998149/35623950-b4e7302e-068d-11e8-9e3c-670e61bb7d01.jpg)

**Comment by curtiola** [comment]
You said you were using steam right?
Looks like it's using Nox as the provider.
I'm pretty sure you just have to:
- open the Yugioh-bot folder
- right click config.ini
- open with notepad
- where it says "provider = Nox" change it to "provider = Steam"
- then save & close

**Comment by faber5594** [comment]
@curtiola nono look, I did it
![immagine](https://user-images.githubusercontent.com/35998149/35624718-10426ba8-0690-11e8-9e0b-96cc790e87a9.jpg)

**Comment by curtiola** [comment]
I don't know then, the bot log you posted says import get_provider... ...import Nox, & then some other Nox related entries. So there must be something else that needs doing. I've never used it on steam, so to me it's a mystery.

**Comment by faber5594** [comment]
ok thanks anyway

**Comment by faber5594** [comment]
@curtiola I managed to solve the problem using the Git Bash instead of cmd, anyway now I have a problem regarding the resolution, seems that it needs a bigger one, but in the configuration app of duel link I don't have such resolution
![immagine](https://user-images.githubusercontent.com/35998149/35634598-59bba9fe-06ac-11e8-8b4e-c9836ce26a6c.jpg)

**Comment by curtiola** [comment]
I'm pretty sure Will is working on a solution for this, I'm not sure when it will be able to support lower resolutions.
The config for Duel Links goes by your monitor's settings I think, can you set it to 1920 x 1080 or are you limited to 768 x 1366?
If you just have it lower for icon size reasons, you can increase the DPI in the display settings:
![image](https://user-images.githubusercontent.com/16383836/35637023-3170a7f6-0670-11e8-8e53-fec4a0ca9a60.png)

**Comment by faber5594** [comment]
@curtiola unforunately I am limited to 1366 x 768, anyway I'm running it on NOX and it seems to work but there some problems mayby due to the language cause it's not englih, now I'm downloading the english version, hope it will work

**Comment by will7200** [comment]
@asdasd888 I will make a video

**Comment by asdasd888** [comment]
thank you

**Comment by dvl9** [comment]
![capture3](https://user-images.githubusercontent.com/37819376/37940271-75e3d310-3170-11e8-817c-67e28f6e08d3.JPG)
![capture2](https://user-images.githubusercontent.com/37819376/37940272-76044172-3170-11e8-8963-80d2ce8946df.JPG)
![capture1](https://user-images.githubusercontent.com/37819376/37940273-762484c8-3170-11e8-92b7-899f4bb70331.JPG)

I followed all the directions and installed the tesseract in the proper folder, but for some reason keep getting a DLL error. Do you have any suggestions for fixing this problem?

**Comment by Reiner6** [comment]
I haven't had any issue during installation, but running the bot i got this problem, anyone had the same issue?
![image](https://user-images.githubusercontent.com/36520286/41244789-66d53934-6d63-11e8-8bb5-d20b9bbdd841.png)
